[blog] 2020 in review and beyond (#24130)
mui-org/material-ui: docs/src/pages/discover-more/roadmap/roadmap.md

- **Better customization.** 💅 We want to make our component customization intuitive, no matter if you are using global CSS or styled-components:
- **Better documentation.** 📚 No problem is solved until documented.
- **Performance.** 🚀 React abstraction has a cost. The more components you render, the slower your page will be. You will notice stark differences when rendering a large table or list.
- **Bundle size.** 📦 You can follow our progress [with bundlephobia.com report](https://bundlephobia.com/result?p=@material-ui/core). Please pay special attention to the cost of the individual modules under "Exports Analysis".
- **TypeScript.** 📏 We are continuously improving the definitions. The codebase is mostly written in JavaScript with manually authored `.d.ts` definitions. While we don't plan a migration effort as as standlone effort, new modules are written in TypeScript.
- **Accessibility.** ♿️ We have relatively [few accessibility issues](https://darekkay.com/blog/accessible-ui-frameworks/), but we are eager to address them all. We would appreciate the help of accessibility experts.

## Quarterly roadmap

> Our GitHub projects roadmap are where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you:

Reviewer: ```suggestion
Our GitHub project's roadmap is where you can learn about what features we're working on, what stage they're at, and when we expect to bring them to you:
```
